ksy_schema.json

```diff
@@ -82,6 +82,151 @@
         "xref": {
           "type": "object",
           "additionalProperties": false,
+          "properties": {
+            "forensicswiki": {
+              "description": "article name at [Forensics Wiki](https://forensicswiki.org/wiki/Main_Page), which is a CC-BY-SA-licensed wiki with information on digital forensics, file formats and tools\n\nfull link name could be generated as `https://forensicswiki.org/wiki/` + this value",
+              "anyOf": [
+                {
+                  "type": "string",
+                  "pattern": "^([a-zA-Z0-9$\\-._~+!*'(),@&;:\\/]+|%[0-9a-fA-F]{2})+$",
+                  "$comment": "hand-crafted regex, it should match any valid URL path"
+                },
```

Comment:
Might be a good idea to extract this into a separate definition (e. g. `MediaWikiPageName`), as we have multiple `xref` keys of this kind (`forensicswiki` and `justsolve`).